src/openms/source/ANALYSIS/XLMS/OPXLHelper.cpp

```diff
@@ -883,79 +887,83 @@ namespace OpenMS
 
       if (top_csms_spectrum[i].cross_link.getType() == OPXLDataStructs::CROSS)
       {
+        ph_alpha.setMetaValue(Constants::OPENPEPXL_BETA_SEQUENCE, (*top_csms_spectrum[i].cross_link.beta).toString());
+
         ph_beta.setSequence(*top_csms_spectrum[i].cross_link.beta);
         ph_beta.setCharge(precursor_charge);
         ph_beta.setScore(top_csms_spectrum[i].score);
-        ph_beta.setRank(DataValue(i+1));
-        ph_alpha.setMetaValue("beta_sequence", (*top_csms_spectrum[i].cross_link.beta).toString());
-        ph_beta.setMetaValue("beta_sequence", (*top_csms_spectrum[i].cross_link.beta).toString());
+        // ph_beta.setRank(DataValue(i+1));
+
+        // TODO when beta PeptideHits are not used anymore, remove most of these, that are not used by PeptideInderer
```

Comment:
there are a lot of commented out stuff
can you remove these (maybe mark them with a commit message so you find them again if needed)